oozie-to-airflow/o2a_libs/helper_functions.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2018 Google LLC
```

Comment:
Can we rename the file to be descriptive of its purpose? Having a generic helper_functions file could result in a dumping grounds in the future. Maybe hdfs_utils.py?